src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/projectnav/ProjectNav.kt

```diff
@@ -0,0 +1,83 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.projectnav
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.beans.property.SimpleStringProperty
+import javafx.geometry.Pos
+import javafx.scene.Node
+import javafx.scene.control.Button
+import javafx.scene.effect.DropShadow
+import javafx.scene.layout.Priority
+import javafx.scene.layout.VBox
+import javafx.scene.paint.Color
+
+import tornadofx.*
+
+class ProjectNav : VBox() {
+    var projectBox: VBox by singleAssign()
+    var chapterBox: VBox by singleAssign()
+    var chunkBox: VBox by singleAssign()
+    val projectIcon = MaterialIconView(MaterialIcon.BOOK, "35px")
+    val chapterIcon = MaterialIconView(MaterialIcon.CHROME_READER_MODE, "35px")
+    val chunkIcon = MaterialIconView(MaterialIcon.BOOKMARK, "35px")
+    val selectProjectTextProperty = SimpleStringProperty()
+    var selectProjectText by selectProjectTextProperty
+
+    val selectChapterTextProperty = SimpleStringProperty()
+    var selectChapterText by selectChapterTextProperty
+
+    val selectChunkTextProperty = SimpleStringProperty()
+    var selectChunkText by selectChunkTextProperty
+
+
+    val nodeList = observableList<Node>()
+
+    fun navbox(mainLabel: String? = null, graphic: Node? = null, init: NavBox.() -> Unit = {}): NavBox {
+        val nb = NavBox(mainLabel, graphic)
+        nb.init()
+        addBox(nb)
```

Comment:
you REALLY don't want to do something like this- the side effect is that when you create a nav box you add it internally to this class. Another developer could come along, use this to create a navbox, and wonder why things are breaking.

The responsibility of adding the box should be separated from the task of creating one.